lib/ruboty/slack_socket_mode/interactive_message.rb

```diff
@@ -46,6 +46,37 @@ def delete
 
         robot.delete_interactive(response_url)
       end
+
+      def update(text: nil, block: nil)
+        response_url = @payload['response_url']
+        unless response_url
+          Ruboty.logger.warn("#{self.class.name}: Cannot update message. This message does not contain response_url in payload.")
+          return
+        end
+
+        if text.nil? && block.nil? || text && block
+          Ruboty.logger.warn("#{self.class.name}: Cannot update message. Wrong number of arguments (expected text or block)")
+          return
+        end
+
+        if text.is_a?(String)
+          robot.update_interactive_message(response_url, text)
+        elsif block.is_a?(Array)
+          robot.update_interactive_block(response_url, block)
```

Comment:
[imo] Rubyではあまり習慣として「引数が想定する型であるか？」の型チェックを記載することはないように思っていて、もし分岐を記載するとしてもこれで良いかなという印象があります〜

```suggestion
        if text
          robot.update_interactive_message(response_url, text)
        else
          robot.update_interactive_block(response_url, block)
        end
```

Reply:
言われるとそうですね。
is_a?をやめてtextとblocksの分岐を無くしました。